app.rb

```diff
@@ -14,13 +18,28 @@ def initialize
   end
 
   def list_all_books
-    puts 'Please add a book to the library' if @books.empty?
-    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    @book = load_data('books.json')
+    if @book.empty?
+      puts 'Please add a book to the library'
+    else
+      data = @book.map(&:to_h)
+      data.each do |book|
+        puts "Title: #{book['title']}, Author: #{book['author']}"
+      end
+    end
   end
 
   def list_all_persons
-    puts 'Please add a person to the library' if @people.empty?
-    @people.each { |person| puts "[#{person.class.name}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    @people = load_data('people.json')
+    if @people.empty?
+      puts 'Please add a person to the library'
+    else
+      data = @people.map(&:to_h)
+      data.each do |person|
+        # puts "[#{person.class.name}] Name: #{person[:name]}, ID: #{person[:id]}, Age: #{person[:age]}"
```

Comment:
- [ ] I think this code is not necessary to keep. Please remove it :)

Reply:
Do you mean only line 39 or the whole from 36-39?
- if you are saying the whole, it will not change to hashes if we remove the else part, 
- please make it clear!
This  will be an error if I remove it :
![image](https://user-images.githubusercontent.com/104406349/226854355-88925a0f-b107-4210-844a-8ebd19d15343.png)

Reply:
I can see in line 39 this commented code: 
```
# puts "[#{person.class.name}] Name: #{person[:name]}, ID: #{person[:id]}, Age: #{person[:age]}"

This one should be removed. If it does not have any necessary action in your code!

Reply:
@Rustamxon7  Thank you for the clarity. 👍

Reply:
This change has been done well